tools/kafka_sasl_ssl/generate_ssl_CA/utils/env_utils

```diff
@@ -0,0 +1,215 @@
+  echo -e "${pass}\n${pass}\ny" | keytool -keystore server.truststore.jks -alias CARoot -import -file ca-root.crt
+
+  if [ $? != 0 ]; then
+    error "Failed to generate certificates."
+    exit 1
+  else
+    prog "Certificates generated successfully."
+  fi
+}
+
+ssl_twoway_ca_generate(){
+  if [ ! -d ${generate_root} ]; then
+    mkdir -p ${generate_root}
+  fi
+
+  cd $generate_root
+  prog "start create ssl_two_way certificates for kafka broker..."
+  keytool -keystore server.keystore.jks -alias ${server_hostname} -validity 365 -genkey -keyalg RSA -dname "cn=${server_hostname}" -storepass ${pass} -keypass ${pass}
+  openssl req -nodes -new -x509 -keyout ca-root.key -out ca-root.crt -days 365 -subj "/C=US/ST=CA/L=Palo Alto/O=Tigergraph/CN=Tigergraph"
+  echo ${pass} | keytool -keystore server.keystore.jks -alias ${server_hostname} -certreq -file ${server_hostname}_server.csr
+  openssl x509 -req -CA ca-root.crt -CAkey ca-root.key -in ${server_hostname}_server.csr -out ${server_hostname}_server.crt -days 365 -CAcreateserial
+  echo ${pass} | keytool -keystore server.keystore.jks -alias CARoot -import -noprompt -file ca-root.crt
+  echo ${pass} | keytool -keystore server.keystore.jks -alias ${server_hostname} -import -file ${server_hostname}_server.crt
+
+  prog "start generate private key, public key certificate pair for the client..."
+  openssl req -newkey rsa:2048 -nodes -keyout ${client_hostname}_client.key -out ${client_hostname}_client.csr -subj "/C=US/ST=CA/L=Palo Alto/O=Tigergraph/CN=Tigergraph"
+  openssl x509 -req -CA ca-root.crt -CAkey ca-root.key -in ${client_hostname}_client.csr -out ${client_hostname}_client.crt -days 365 -CAcreateserial
+  echo -e "${pass}\n${pass}\ny" | keytool -keystore server.truststore.jks -alias CARoot -import -file ca-root.crt
+
+  if [ $? != 0 ]; then
+    error "Failed to generate certificates."
+    exit 1
+  else
+    prog "Certificates generated successfully."
+  fi
+}
+
+help(){
+  echo
+  echo "Usage:"
+  mesg_green "./`basename $0` [-o ssl_two_way] [-c <tigergraph_client_name>] [-s <broker_host_name>] [-t <ssl_file_path>] [-p <password>] [-i]"
+  mesg_green "./`basename $0` [-o ssl_two_way] [-c <tigergraph_client_name>] [-s <broker_host_name>] [-t <ssl_file_path>] [-p <password>]"
+  mesg_green "./`basename $0` [-o ssl_one_way] [-s <broker_host_name>] [-t <ssl_file_path>] [-p <password>] [-i]"
+  mesg_green "./`basename $0` [-o ssl_one_way] [-s <broker_host_name>] [-t <ssl_file_path>] [-p <password>]"
+  mesg_green "./`basename $0` -h"
+  echo "Options:"
+  echo "  -h  --  Show the help"
+  echo "  -o  --  SSL certificate type: ssl_one_way or ssl_two_way [default: ssl_one_way]"
+  echo "  -s  --  Kafka broker host name [default: kafka-0.tigergraph.com]"
+  echo "  -t  --  Certificate Generation Path [default: ~/SSL_files]"
+  echo "  -c  --  Tigergraph Client host name"
+  echo "  -p  --  Password [default: tiger123]"
+  echo "  -i  --  Install openjdk-1.8.0"
```

Comment:
please change the note:
`-i -- Install openjdk-1.8.0 and openssl`

If user don't add `-i`, and jdk-1.8 or openssl is missing, report error